src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,193 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K,V>[] table;
+
+    private int size;
+
```

Comment:
```suggestion
    private static final int DEFAULT_INITIAL_CAPACITY = 16;
    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
    private Node<K,V>[] table;
    private int size;
```
 поля не потрібно розділяти між собою пустими строками